core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -768,7 +867,8 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
 
   private def createNewWorkspaceRecord(workspaceId: UUID,
                                        request: WorkspaceRequest,
-                                       parentContext: RawlsRequestContext
+                                       parentContext: RawlsRequestContext,
+                                       state: WorkspaceState = WorkspaceState.Ready
```

Comment:
I had to add the option to specify a different state, so we could create it as `Cloning`.
Defaults to `Ready` so I didn't have to fuss with changing existing calls.